src/libs/actions/Report.js

```diff
@@ -40,15 +41,7 @@ Onyx.connect({
 let preferredSkinTone;
 Onyx.connect({
     key: ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE,
-    callback: (val) => {
-        // the preferred skin tone is sometimes still "default", although it
-        // was changed that "default" has become -1.
-        if (!_.isNull(val) && Number.isInteger(Number(val))) {
-            preferredSkinTone = val;
-        } else {
-            preferredSkinTone = -1;
-        }
-    },
+    callback: EmojiUtils.getPreferredSkinToneIndex,
```

Comment:
@tienifr Is `preferredSkinTone` not taking any value from Onyx? `EmojiUtils.getPreferredSkinToneIndex` is returning a number, shouldn't we assign the value to `preferredSkinTone`?